src/useObservableEffect.ts

```diff
@@ -21,13 +21,14 @@ export function useObservableEffect<D extends IReactionDisposer>(
         disposerRef.current = disposerGenerator()
     }, inputs)
 
-    useEffect(() => {
-        return () => {
+    useEffect(
+        () => () => {
```

Comment:
Shoot, Prettier let us down here and indentation is still the same. Oh well :) Machine owns us.

Reply:
Now you start to see why I added that optional inputs to useDismount :)